docs/Frameworks.md

```diff
@@ -51,7 +51,7 @@ it('should test something', () => {
 })
 ```
 
-If you want to run something asynchronously, you can either use the [`browser.call`](call) command or [custom commands](CustomCommands.md).
+If you want to run something asynchronously, you can either use the [`browser.call`](https://webdriver.io/docs/api/browser/call.html) command or [custom commands](CustomCommands.md).
```

Comment:
```suggestion
If you want to run something asynchronously, you can either use the [`browser.call`](/docs/api/browser/call.html) command or [custom commands](CustomCommands.md).
```

Reply:
maybe
```suggestion
If you want to run something asynchronously, you can either use the [`browser.call`](/api/browser/call.md) command or [custom commands](CustomCommands.md).
```

Reply:
There is no `browser` directory and the `call` file at all :(

Reply:
@elaichenkov this file is autogenerated if you call `npm run docs:generate`